torch/_dynamo/variables/builder.py

```diff
@@ -1141,9 +1179,18 @@ def _clone_input(value):
     elif proxy.node.target in [torch.cuda.streams.Stream, torch.cuda.current_stream]:
         proxy.node.meta["example_value"] = example_value
         return CUDAStreamVariable(proxy, example_value, **options)
-    elif config.numpy_ndarray_as_tensor and isinstance(example_value, torch_np.ndarray):
-        proxy.node.meta["example_value"] = example_value
-        return target_cls(proxy, **options)
+    elif config.numpy_ndarray_as_tensor:
```

Comment:
So, the immediate bug here is that you have a nested if-block, but this means you never get to the else block at the end.

But in general, the change you've made here seems pretty questionable. What are you actually trying to do here?  It seems related to the fact that you're using `wrap_fx_proxy` to handle size/stride calls. But there is already handling for int/etc returns (though guarded on dynamic shapes). Perhaps it would be simplest to just gate numpy ndarray on dynamic shapes? (That will be turned on by default soon, btw, thanks to @voznesenskym work)